src/EditorFeatures/CSharp/AutomaticCompletion/AutomaticLineEnderCommandHandler.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.AutomaticCompletion
     /// <summary>
     /// csharp automatic line ender command handler
     /// </summary>
-    [ExportCommandHandler(PredefinedCommandHandlerNames.AutomaticLineEnder, ContentTypeNames.CSharpContentType)]
+    [ExportRoslynCommandHandlerAttribute(PredefinedCommandHandlerNames.AutomaticLineEnder, ContentTypeNames.CSharpContentType)]
```

Comment:
:bulb: `Attribute` suffix is unnecessary (applies many places).

Reply:
Ah, fun bug in inline rename :)

Reply:
Did you file a bug for it?

Reply:
It's https://github.com/dotnet/roslyn/issues/21657